.github/CODEOWNERS

```diff
@@ -160,6 +160,9 @@
 # PRLabel: %Open AI
 /sdk/openai/azure-ai-openai/                         @brandom-msft @jpalvarezl @mssfang
 
+# PRLabel: %Image Analysis
+/sdk/vision/azure-ai-vision-imageanalysis/           @dargilco @rhurey
```

Comment:
This label did not exist; I've created it with the proper configuration for our automation.

This line covers pull requests, but we'll also want to add an entry near the bottom of the file (looks like they start around L287) that defines who to contact for issues that we tag for service attention.   Assuming the same contacts, that would look like:

```text
# ServiceLabel: %Image Analysis %Service Attention
/sdk/vision/azure-ai-vision-imageanalysis/           @dargilco @rhurey
```

Reply:
Thank you @jsquire for adding the `Image Analysis` label, and the comment above! I will make this change in the next commit to this PR.